concrete-core/Cargo.toml

```diff
@@ -40,6 +40,7 @@ doc = []
 backend_core = []
 slow-csprng = ["concrete-csprng/slow"]
 multithread = ["rayon", "concrete-csprng/multithread"]
+serde_serialization = ["serde", "concrete-commons/serde"]
```

Comment:
Here you should add `concrete-fftw/serialize` as a feature dependency, and remove it from line 24 I think.